tests/e2e/scripts/hermes_bootstrap.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
please use go-relayer instead of hermes relayer. Go relayer receives new changes and experiments from ibc-go specs and is easier to debug in golang and test.

Reply:
Ibc-go is using hermes as default relayer. i don't see any necessity to use go relayer instead of hermes
https://github.com/cosmos/ibc-go/blob/9a00ffef9f0f364f6864126e138990df6c43c536/e2e/README.md#L58

Reply:
What you mean by go relayer receives experiment from ibc-go specs ?

Reply:
ibc query was integrated onto go - relayer during its experimental so I have been using go - relayer for the development of fee - abstraction and other modules. This also works really well when I face issues and need to debug within go - relayer.